README.md

```diff
@@ -21,53 +21,44 @@ in instances such as cluster deletion or disaster.
 
 <hr style="height:1px;border:none;color:#333;">
 
-To use the data mover controller, you will first need a volumeSnapshot. This can be achieved
-by using the Velero CSI plugin during backup of the stateful application.
-
-- Install OADP Operator using [these steps](https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md).
-
 - Have a stateful application running in a separate namespace. 
 
-- [Install](https://volsync.readthedocs.io/en/stable/installation/index.html) the VolSync controller.
+- Have an appropriate StorageClass and VolumeShapshotClass:
```

Comment:
Can we something regarding the cluster having only one `default` volumesnapshotclass ?